Userland/Libraries/LibGfx/Bitmap.h

```diff
@@ -23,7 +23,8 @@
     __ENUMERATE_IMAGE_FORMAT(ico, ".ico")  \
     __ENUMERATE_IMAGE_FORMAT(jpg, ".jpg")  \
     __ENUMERATE_IMAGE_FORMAT(jpg, ".jpeg") \
-    __ENUMERATE_IMAGE_FORMAT(dds, ".dds")
+    __ENUMERATE_IMAGE_FORMAT(dds, ".dds")  \
+    __ENUMERATE_IMAGE_FORMAT(dds, ".qoi")
```

Comment:
I think this might a copy paste oversight and should be the following?
```suggestion
    __ENUMERATE_IMAGE_FORMAT(qoi, ".qoi")
```

Reply:
Oops, fixed!